letsdrill.rb

```diff
@@ -1,24 +1,41 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  result = ""
+  if integer > 89
+    result = "A"
+  elsif integer > 79
+    result = "B"
+  elsif integer > 69
+    result = "C"
+  elsif integer > 59
+    result = "D"
+  else
+    result = "F"
+  end
 
+  return result
```

Comment:
A couple things here:
 - does this work without initializing `result`?
 - standard ruby styling is to omit `return` unless you're doing an early return
 - how about a case statement here?